jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/HttpInput.java

```diff
@@ -257,6 +277,16 @@ public int read() throws IOException
 
     @Override
     public int read(byte[] b, int off, int len) throws IOException
+    {
+        return read(null, b, off, len);
+    }
+
+    public int read(ByteBuffer buffer) throws IOException
```

Comment:
Does anything use this method?

Reply:
Yeah, I checked out your branch for this PR and checked usages.
Nothing uses this method.